api/graphql/index.js

```diff
@@ -239,12 +241,14 @@ export function makeMutations (sessionId, userId, isAdmin) {
 
     createSavedSearch: (root, { data }) => createSavedSearch(data),
 
-    joinGroup: (root, {groupId}) => joinGroup(groupId, userId),
+    joinGroup: (root, { groupId }) => joinGroup(groupId, userId),
 
     joinProject: (root, { id }) => joinProject(id, userId),
 
     createTopic: (root, { topicName, groupId, isDefault, isSubscribing }) => createTopic(userId, topicName, groupId, isDefault, isSubscribing),
 
+    deactivateMe: (root, { id }) => deactivateUser({ sessionId, userId: id }),
```

Comment:
i just noticed that you are passing in id here, shouldnt you just always use userId? we dont want to let people deactivate other users, at least through this deactivateMe call